commitlint.config.js

```diff
@@ -24,5 +24,17 @@ module.exports = {
       'always',
       validateBodyMaxLengthIgnoringDeps,
     ],
+    'function-rules/body-jira-id-check': [
+      2, // level: error
+      'always',
+      (parsed) => {
+        const jiraRegex = parsed.body.match(/[A-Z]+[-\d]+/g);
+        const jiraIgnoreRegex = parsed.body.match(/I don’t have a ticket for this/g);
```

Comment:
`TECH` will be the keyword for "I don’t have a ticket for this" 👍 

```suggestion
        const jiraIgnoreRegex = parsed.body.match(/TECH/g);
```